packages/dendron-next-server/pages/vscode/note-preview.tsx

```diff
@@ -1,21 +1,47 @@
-import { DMessageSource, NoteViewMessageType } from "@dendronhq/common-all";
+import {
+  DendronConfig,
+  DMessageSource,
+  NoteViewMessageType,
+} from "@dendronhq/common-all";
 import {
   createLogger,
   engineHooks,
   engineSlice,
   postVSCodeMessage,
 } from "@dendronhq/common-frontend";
 import { Col, Layout, Row } from "antd";
+import _ from "lodash";
 import * as React from "react";
+import { useThemeSwitcher } from "react-css-theme-switcher";
 import { getWsAndPort } from "../../lib/env";
 import { DendronProps } from "../../lib/types";
+import { getThemeType, ThemeType } from "../../styles/theme";
 
 const logger = createLogger("notePreview");
 
 function isHTMLAnchorElement(element: Element): element is HTMLAnchorElement {
   return element.nodeName === "A";
 }
 
+type MermaidInitialzeParams = {
+  startOnLoad: boolean;
+  cloneCssStyles: boolean;
+  theme: string;
+};
+
+type Mermaid = {
```

Comment:
Should we use this instead? https://www.npmjs.com/package/@types/mermaid

Reply:
unfortunately that's a type declaration file. they attach themselves to the Mermaid library but its not straightforward to use them without importing the library (which we don't do because we're currently using the CDN version)